docs-ref-services/servicebus.md

```diff
@@ -15,34 +15,189 @@ ms.service: service-bus
 
 ## Overview
 
-Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging. 
+Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging.
+
+As of version 0.50.0 a new AMQP-based API is available for sending and receiving messages. This update involves breaking changes.
+Please read [Migration from v0.21.1 to v0.50.0](#migration-from-v0.21.1-to-v0.50.0) to determine if upgrading is
+right for you at this time.
+
+For documentation on the legacy HTTP-based operations please see [Using HTTP-based operations of the legacy API](#using-http-based-operations-of-the-legacy-api)
+
+
+## Prerequisites
+
+* Azure subscription - [Create a free account](https://azure.microsoft.com/free/)
+* Azure [Service Bus namespace and management credentials](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-create-namespace-portal)
+* [Python 2.7-3.7](https://www.python.org/downloads/)
+
 
 ## Install the libraries
 ```bash
 pip install azure-servicebus
 ```
 
+## Create Client
+
+To get started, you will need to create a ServiceBusClient using the
+namespace and credentials that can be retrieved from the Azure Portal.
+```python
+import os
+from azure.servicebus import ServiceBusClient
+
+namespace = os.environ['SB_NAMESPACE']
+policy = os.environ['SB_ACCESS_POLICY']
+key = os.environ['SB_ACCESS_KEY']
+
+sb_client = ServiceBusClient(
+    service_namespace=namespace,
+    shared_access_key_name=policy,
+    shared_access_key_value=key)
+```
+A client can also be created with the Connection String from the portal directly:
```

Comment:
We've noticed that presenting only one way to do something is typically better than giving options. If possible, stick with the easiest/most commonly used method, and you can ditch the other.

If you *do* need to specify both, add a line telling the customer why they'd choose one over the other.

Reply:
Thanks - I've removed the first option and kept the "via connection string" option.
Getting the connection string from the portal seems to be the easiest way to connect and is what we found most people using EventHubs were using.
Getting the connection string is also what is described in the "Create Service Bus" tutorial I linked to in prerequisites.